### PR TITLE
Mvc identity

### PR DESCRIPTION
**Pull request by AntsiferauGodel**
Identity, users, roles, prep for advertisements

### CODE REVIEW

File: Lab08.MVC/Lab08.MVC.Business/Services/ServiceCreator.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using Lab08.MVC.Business.Interfaces;
+using Lab08.MVC.Data.Uows;
+
+namespace Lab08.MVC.Business.Services
+{
+    public class ServiceCreator : IServiceCreator
+    {
+        public IUserService CreateUserService(string connectionString)
+        {
```
[the remark this answers is not in the document]
Comment (AntsiferauGodel) [reply Lab08.MVC/Lab08.MVC.Business/Services/ServiceCreator.cs]: yes, in фгещафс branch there is no more serviceCreator

File: Lab08.MVC/Lab08.MVC.Business/Models/LoginModel.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Lab08.MVC.Business.Models
+{
+    public class LoginModel
+    {
```
[the remark this answers is not in the document]
Comment (AntsiferauGodel) [reply Lab08.MVC/Lab08.MVC.Business/Models/LoginModel.cs]: saw that in example https://github.com/pryabov/Education-Materials/tree/trivial-mvc-example/trivial-mvc-architecture

File: Lab08.MVC/Lab08.MVC.Data/DbContexts/Initializators/FleaMarketInitializator.cs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                applicationUsers.Add(new ApplicationUser
+                {
+                    Email = firstRole + i + mailDomain,
+                    UserName = firstRole + i + mailDomain
+                });
+            }
+            var roles = new List<string>
+            {
+                "trader",
+                "buyer"
```
[the remark this answers is not in the document]
Comment (AntsiferauGodel) [reply Lab08.MVC/Lab08.MVC.Data/DbContexts/Initializators/FleaMarketInitializator.cs]: added

File: Lab08.MVC/Lab08.MVC.Business/Models/LoginModel.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (HannaKeryna) [reply Lab08.MVC/Lab08.MVC.Business/Models/LoginModel.cs]: I want to know you opinion and reason. I can find lots of examples

**Pull request merged**